app/lib/scraper.rb

```diff
@@ -0,0 +1,62 @@
+class Scraper
+
+
+
+  def self.fetch
+
+
+    journal_feeds = JournalFeed.all
+
+    feed = Feedjira::Feed.fetch_and_parse(journal_feeds.last.url)
+    binding.pry
+    journal_feeds.each do |journal_feed|
+      feed = Feedjira::Feed.fetch_and_parse(journal_feed.url)
+      journal = Journal.new({journal_feed: journal_feed, title: journal_feed.title, date: feed.entries.first.published })
+      unless Scraper.dates(journal_feed).include?(journal.date)
```

Comment:
This is a great first run of making sure you're only loading new journals.That's always the first step, to make it work any way you can. But once it's working you can start thinking about ways to improve the speed, readability or sim0licity of the code. 

Reading this line I can understand what it's doing after a bit of though, but it's not obvious at a glance. See if you can come up with a way to refactor this check to make it more understandable. The easiest way to do that is to create a method that executes the code, and give the method a name that says what it's doing. You could also look at the `break` keyword and see if it would be handy here.